### ISSUE TITLE
Some imported files produce ugly error message when trying to print

### ISSUE THREAD

**Issue by oskay** [evil-mad/axidraw#24]
If units are not found, it should be handled gracefully with an error message-- possibly indicating that the file should be opened with File > Import.

**Comment by oskay** [comment]
Example of the error:
```
Traceback (most recent call last):
  File "axidraw.py", line 2076, in <module>
    e.affect()
  File "/Applications/Inkscape.app/Contents/Resources/share/inkscape/extensions/inkex.py", line 265, in affect
    self.getposinlayer()
  File "/Applications/Inkscape.app/Contents/Resources/share/inkscape/extensions/inkex.py", line 207, in getposinlayer
    x = self.unittouu( xattr[0] + 'px' )
  File "/Applications/Inkscape.app/Contents/Resources/share/inkscape/extensions/inkex.py", line 351, in unittouu
    return retval * (self.__uuconv[u.string[u.start():u.end()]] / self.__uuconv[self.getDocumentUnit()])
  File "/Applications/Inkscape.app/Contents/Resources/share/inkscape/extensions/inkex.py", line 304, in getDocumentUnit
    p = param.match(svgwidth)
TypeError: expected string or buffer
```

Document producing the error attached.

[ErrorDocument.zip](https://github.com/evil-mad/axidraw/files/715282/ErrorDocument.zip)

**Comment by oskay** [comment]
After some digging, it turns out that the error occurs in the execution of the Inkscape extension _before_ the code for the extension itself is run. 

It's an Inkscape bug, fixed as of Inkscape 0.92: 
https://bugs.launchpad.net/inkscape/+bug/1463623
